### PR TITLE
Fixes and Updates

### PR DESCRIPTION
**Pull request by EmberSkyMedia**
Some Typo Fixes and Connected living space update for parachute handle.

### CODE REVIEW

**Comment by zorg2044** [comment]
Please don't remove the ``?`` from the tags. its not a typo. It limits the tag to exact matches, its good practice particularly for very short tags like BCS.

File: GameData/BoringCrewServices/Parts/Starliner/BCS_Centauri_LAE.cfg [hunk 1]
```diff
@@ -45,7 +45,7 @@ PART
 	maxTemp = 2000 // = 3600
 	bulkheadProfiles = size0, srf
 
-	tags =  Starliner Boring ?BCS engine RS88 Abort
+	tags =  Starliner Boring BCS engine RS88 Abort
```
Comment (SofieBrink) [inline GameData/BoringCrewServices/Parts/Starliner/BCS_Centauri_LAE.cfg:48]: as Zorg said [here](https://github.com/zorg2044/BoringCrewServices/pull/28#issuecomment-2408606183)

File: GameData/BoringCrewServices/Patches/ConnectedLivingSpaces.cfg [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	}
 }
 
-@part[BCS_Centauri_CrewCapsule]:NEEDS[ConnectedLivingSpace]:FOR[BoringCrewServices]
+@part[BCS_Centauri_CrewCapsule|BCS_Centauri_MainChute|BCS_Centauri_DrogueChutes]:NEEDS[ConnectedLivingSpace]:FOR[BoringCrewServices]
```
Comment (SofieBrink) [inline GameData/BoringCrewServices/Patches/ConnectedLivingSpaces.cfg:10]: This feels unnececary per the same reason as discussed at the start [here](https://github.com/zorg2044/BoringCrewServices/issues/5#issuecomment-2409099311)

**Review by SofieBrink — CHANGES_REQUESTED** [review]

File: GameData/BoringCrewServices/Patches/ConnectedLivingSpaces.cfg (hunk 2 above)
Comment (EmberSkyMedia) [reply GameData/BoringCrewServices/Patches/ConnectedLivingSpaces.cfg:10]: Closed the request

File: GameData/BoringCrewServices/Parts/Starliner/BCS_Centauri_LAE.cfg (hunk 1 above)
Comment (EmberSkyMedia) [reply GameData/BoringCrewServices/Parts/Starliner/BCS_Centauri_LAE.cfg:48]: Closed the pull request

**Pull request closed without merging**